apis/monitoring/v1beta1/monitoringdashboard_types.go

```diff
@@ -300,13 +297,8 @@ type CollapsibleGroup struct {
 
 // +kcc:proto=google.monitoring.dashboard.v1.ErrorReportingPanel
 type ErrorReportingPanel struct {
-	// The resource name of the Google Cloud Platform project. Written
-	//  as `projects/{projectID}` or `projects/{projectNumber}`, where
-	//  `{projectID}` and `{projectNumber}` can be found in the
-	//  [Google Cloud console](https://support.google.com/cloud/answer/6158840).
-	//
-	//  Examples: `projects/my-project-123`, `projects/5551234`.
-	ProjectNames []string `json:"projectNames,omitempty"`
+	// The projects from which to gather errors.
+	ProjectRefs []refs.ProjectRef `json:"projectRefs,omitempty"`
```

Comment:
Do we need this to be a pointer (for deepcopy and other pointer reference methods)?
```
type ErrorReportingPanel struct {
	ProjectRefs *ProjectRefList ...
}
type ProjectRefList struct {
        []*refs.ProjectRef
}
```

Reply:
If so, maybe we can move this to https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/apis/refs/v1beta1/projectref.go so it will be unified.

Reply:
We discussed, I think what we have now is what we agreed:

```
      - title: "ErrorReporting Widget"
        errorReportingPanel:
          projectRefs:
          - external: projects/project1
          - external: projects/project2
          services:
          - foo
          - bar
          versions:
          - v1
          - v2
```